docs/ops_heroku_settings.md

```diff
@@ -0,0 +1,36 @@
+# OPS and Heroku Settings
+
+## Staging
+
+Builds to staging are triggered by commits to `master`.
+
+Staging URL is [foundation.mofostaging.net](https://foundation.mofostaging.net)
+
+## Production
+
+Production deployments are done by promoting Staging in the Heroku pipeline.
+
+Production URL is [foundation.mozilla.org](https://foundation.mozilla.org)
+
+## Domain Redirect
+
+Enable domain redirection by setting `DOMAIN_REDIRECT_MIDDLEWARE_ENABLED` to `True`. This will enable a middleware function that checks every request, and return a 307 redirect to the first listed domain in `TARGET_DOMAINS` if the host header does not match one of the domains specified there.
+
+## Special purposes Environment Variables
+
+- `GITHUB_TOKEN`: GITHUB API authentication,
+- `SLACK_WEBHOOK_RA`: Webhook to `mofo-review-apps`
+- `CORAL_TALK_SERVER_URL`: If Coral Talk commenting is to be enabled, set the server URL here. Don't forget to add the domain to your CSP directives for script-src and child-src
+
+
+## Cloudinary for Review Apps and Staging (BuyersGuide only)
+
+We use Cloudinary upload-mapping feature to copy images from the production to the staging Cloudinary account.
```

Comment:
```suggestion
We use Cloudinary's upload-mapping feature to copy images from the production to the staging Cloudinary account.
```
Maybe with a link for "upload-mapping" that points to the page that explains what that is?